### DISCUSSION TITLE
Setting cron to 1 min

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#14405] [General]
Anyone got some bootle-neck effect when setting crontab to 1 min? Need to send workflow email every 1 min, but that depending on crontab seems like. Appreciate pointers

**Comment by vovpff** [comment]
I use 1 min cron two years. Email cron also 1 min. I can't feel any bads. But my system just for 15 users and email messages amount not so big

**Comment by mariuszkrzaczkowski** [comment]
it can be every 1 min, it all depends on the number of CPU threads
